src/modules/VirtConfig.rb

```diff
@@ -235,6 +235,7 @@ def Information
                       VBox(
                         Left(Label(_("Server: Minimal system to get a running Hypervisor"))),
                         Left(Label(_("Tools: Configure, manage and monitor virtual machines"))),
+                        Left(Label(_("A desensitized checkbox means the Hypervisor item has already been installed"))),
```

Comment:
Um, we usually use word `disabled` for grayed out (inactive) widgets. An reason to use `desensitized` here?